API.md

```diff
@@ -72,7 +72,11 @@ The next metadata keys are usually filled
   - name - A name for the skin
   - author - The skin author
   - license - THe skin texture license
-  - assignment  - is "player:playername" in case the skin is assigned to be privat for a player
+  - assignment - (obsolete) is "player:playername" in case the skin is assigned to be privat for a player
```

Comment:
Why deprecate `assignment`? What would make use for NPCs possible, like the assignments `npc_trader:trader_3` or `npc_warrior:hulk` in future versions.

Reply:
I do not like the assignment for 2 reasons:
String operation: the playername cannot be just used
Only 1 assignment for skin possible. I like to introduce multi-assignment. We have meta that allows different skin attributes, so I do not like to force the "assignment" attribute anymore.

By default all skins are usable for all players. If you like to use it for NPC's you can set "playername" attribute to false and add custom metadata like "npc_type".

I see the describe solution is still incomplete. I plan to implement next things if I am back on my PC:
1. allow playername == false (change check to "if playername ~= nil")
2. Add new function skins.get_skinlist_with_meta(key, value)

Reply:
For the question about difference between  `skins.get_skinlist_for_player(playername)` and ` skins.get_skinlist_with_meta('playername', playername)`: THe first one gets public and private skins, the second gets the private skins only,

Reply:
All done